docs/source/model_design/risk_factors.rst

```diff
@@ -195,7 +195,118 @@ relationships directionally between a cause/exposure and effect/outcome.
   Discuss criteria (sim science and GBD) for modeling a risk outcome pair
 
 Confounding
-""""""""""""
+"""""""""""
+**What is confounding?**
+
+Much of epidemiology is concerned with establishing associations between exposures and health outcomes. This usually involves comparing representative groups from two populations that are as similar as possible in all respects except the exposure(s) under study. **Confounding occurs when an estimate of the association between an exposure and a health outcome is mixed up with the effect of another exposure on the same health outcome, and the two exposures are associated**. When present, it results in a biased estimate of the effect of exposure on the health outcome. The bias can be negative (resulting in underestimation of the exposure effect), or positive (overestimation of the exposure effect), or can even reverse the apparent direction of effect. Confounding should always be addressed when assessing causality and failure to account for confounding leads to incorrectly concluding that an effect or its magnitude is due to one, rather than another variable. 
```

Comment:
I think you can still winnow this down a bit, and start right off with the definition in this first sentence. is there a way to state it without using 'mixed up'?

Reply:
mixed together? I don't want to use the word 'confound' in the definition.